index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <div class="container">
+        <!--TOOLBAR SECTION-->
+        <nav class="toolbar">
```

Comment:
Kindly update this element to use a `<header></header>` tag instead of `<nav class='toolbar'`🙏. The header tag is used for semantic reasons as it helps describe what that section of the page is all about. Also it improves your page SEO results